Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -229,14 +229,10 @@ class StringSetting: Setting, UITextFieldDelegate {
         textField.placeholder = placeholder
         textField.delegate = self
         textField.addTarget(self, action: #selector(textFieldDidChange), forControlEvents: .EditingChanged)
-
-        cell.accessibilityTraits = UIAccessibilityTraitLink
+        cell.userInteractionEnabled = true
+        cell.accessibilityTraits = UIAccessibilityTraitNone
         cell.contentView.addSubview(textField)
 
-        let container = UIView()
-        container.addSubview(textField)
-        cell.contentView.addSubview(container)
-
```

Comment:
Hm, I wonder what this `container` was originally supposed to be for? Kinda weird that we do `cell.contentView.addSubview(textField)`, then `container.addSubview(textField)` right below with the same `textField`.